version.bzl

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 """The version of rules_pkg."""
 
-version = "0.5.1"
+version = "0.6.0"
```

Comment:
Probably worth mentioning this in the commit message.

Also, the distro package is a [tarbomb](https://en.wiktionary.org/wiki/tarbomb) (it was before) -- should we put a prefix directory inside it?

Reply:
Done